docs/content/new-tutorial/intro-to-assets.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introduction to the Dagster Tutorial and Assets | Dagster Docs"
+description: "Learn about the objectives for the Dagster tutorial and about assets"
+noindex: True
+---
+
+# Introduction to Dagster
+
+Dagster is an orchestration framework to help data engineers deliver high-quality data. In Dagster, the objects made by data engineers are called assets.
+
+In this tutorial, you will write a data pipeline that performs ETL (Extract, Transform, Load). You will:
```

Comment:
Instead, I think we could talk more specifically about what we're aiming to accomplish here and why.  E.g. "We want to track and understand activity on a popular news aggregation website. To do so, we'll build a dataset that's a cleaned-up version of the events from the site, and we'll build a report on top of that dataset that summarizes it. We'll then set up Dagster to keep these assets – the dataset and the report – up-to-date over time".

Reply:
😃 I really like the framing you have in your second comment. I think I was trying to hold off on the narrative until they started writing code, but you're absolutely right that it actually deserves to be in the intro. Definitely going to steal that you just wrote though, ty!

💭 re: ETL, yeeaeaaaahh, you can tell I kind of shoe-horned the concept of ETL into the steps that are being done. Because you're right, it's _not_ ETL. The reason I framed it through ETL was that I've been seeing lots of feedback that Dagster/assets are too different from what they know and requires a large shift in perspective.

💭 The ETL portion wasn't for people who don't know ETL, but rather to enable those who do know it to anchor their learning into something they already know. Also,that there are assets in each stage (though as you mentioned, the mapping in this example isn't great). Tough balance to strike.

What's your take on it?

Reply:
My observation has been that, even though people are familiar with the acronym ETL, it's not a very concrete concept in their minds.  I think it's a reasonable stand-in for "data pipelines" in a lot of contexts, but when I try to think about the specifics, I always end up scratching my head. The paradigm that most people use is more like ELTLTLTLTLTL - i.e. they extract data frame somewhere, load it without transforming it, then transform it and load the transformed thing, transform that and load the transformed thing, etc.

Reply:
> The reason I framed it through ETL was that I've been seeing lots of feedback that Dagster/assets are too different from what they know and requires a large shift in perspective.

I think this is very legitimate, but I think a good middle ground is to talk about building and maintaining datasets, which is something that's not a Dagster concept but is also not too jargon-y.